### ISSUE TITLE
Couldn't find module 

### ISSUE THREAD

**Issue by Moulay92** [adampax/titanium-amplitude#1]
I use 5.5.1.GA on my project, I get this error when I tried to use your Module "Couldn't find module: com.polancomedia.amplitude for architecture: x86_64".
please I need Help.

**Comment by adampax** [comment]
@Moulay92 Yes, these modules need to be recompiled with 64-bit support.  I'd be happy to accept a PR from you with the updated module!

**Comment by Moulay92** [comment]
So Sorry @adampax, I would like to do it but i dont know the method. please I need the update
